pyaedt/modeler/MultiPartComponent.py

```diff
@@ -0,0 +1,1183 @@
+import json
+from glob import glob
+import os
+import math
+import sys
+from pyaedt.generic.general_methods import aedt_exception_handler
+
+
+def get_numeric(s):  # TODO: add validation
```

Comment:
```suggestion
def _get_numeric(s):  # TODO: add validation
```
I would put all static methods as private. I think they are used only inside this module.

Reply:
I'll change it accordingly on every point is used